lstn/static/js/controllers.js

```diff
@@ -383,6 +383,7 @@ angular.module('lstn.controllers', [])
       }
 
       // Play the track through Rdio
+      apiswf.rdio_setVolume(0.7);
```

Comment:
Unless I'm missing something, we don't seem to be providing a way to adjust the volume in the app. Is there a reason why we're tweaking it here?